integration/setup/common.go

```diff
@@ -39,14 +45,24 @@ func common(config Config) error {
 			return microerror.Mask(err)
 		}
 
-		err = config.Host.InstallStableOperator("cert-operator", "certconfig", values)
+		err = config.Release.InstallOperator(ctx, key.ReleaseName("cert-operator"), release.NewStableVersion(), values, corev1alpha1.NewCertConfigCRD())
```

Comment:
This is now really ugly. We need to namespace three times. 

* The values above need a Kubernetes namespace so Helm installs the manifests correctly where we want them to be. 
* The release name here has to be prefixed because we install operators multiple times. I used the target namespace here now as well. 
* The release package itself though needs to be configured with the target namespace because it does the pod existence check implicitly internally. This is weird and as I already discussed with Pawel this interface here should just take a pod existence condition and not do magic internally. The release service implementation then maybe does not have to be configured with a target namespace at all.

Reply:
And now I had to prefix it internally because changing the name here fucks with the chart names.